src/styles/main.scss

```diff
@@ -6,6 +6,7 @@ html {
     font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
     font-size: 62.5%;
     font-weight: 300;
+    line-height: 1.42857143;
```

Comment:
No units is allowed for line-height.  From the spec, if you only include a unitless number it `will be multiplied with the current font size to set the line height`.

That's actually what we want, since it allows it to scale properly with font-size changes.